public/app/features/manage-dashboards/components/ImportDashboardForm.tsx

```diff
@@ -0,0 +1,185 @@
+                  target="_blank"
+                >
+                  Grafana.com
+                </a>
+              </Forms.Legend>
+            </div>
+            <table className="filter-table form-inline">
+              <tbody>
+                <tr>
+                  <td>Published by</td>
+                  <td>{meta.orgName}</td>
+                </tr>
+                <tr>
+                  <td>Updated on</td>
+                  <td>{dateTime(meta.updatedAt).format('YYYY-MM-DD HH:mm:ss')}</td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+        )}
+        <Forms.Form onSubmit={this.onSubmit}>
+          {() => {
+            return (
+              <>
+                <Forms.Legend className="section-heading">Options</Forms.Legend>
+                <Forms.Field label="Name" invalid={nameExists} error={nameError}>
+                  <Forms.Input size="md" type="text" value={dashboard.title} onChange={this.onTitleChange} />
+                </Forms.Field>
+                <Forms.Field label="Folder">
+                  <FolderPicker onChange={this.onFolderChange} useInNextGenForms={true} initialFolderId={0} />
+                </Forms.Field>
+                <Forms.Field
+                  label="Unique identifier (uid)"
+                  description="The unique identifier (uid) of a dashboard can be used for uniquely identify a dashboard between multiple Grafana installs.
+                The uid allows having consistent URL’s for accessing dashboards so changing the title of a dashboard will not break any
+                bookmarked links to that dashboard."
+                  invalid={uidExists}
+                  error={uidError}
+                >
+                  <Forms.Input
+                    size="md"
+                    value={uidReset ? dashboard.uid : 'Value set'}
+                    onChange={this.onUidChange}
+                    disabled={!uidReset}
+                    addonAfter={<Forms.Button onClick={this.onUidReset}>Clear</Forms.Button>}
+                  />
+                </Forms.Field>
+                {inputs.map((input: any, index: number) => {
+                  if (input.type === 'datasource') {
+                    return (
+                      <Forms.Field label={input.label} key={`${input.label}-${index}`}>
+                        <DataSourcePicker
+                          datasources={input.options}
+                          onChange={() => console.log('something changed')}
+                          current={input.options[0]}
+                        />
+                      </Forms.Field>
+                    );
+                  }
+                  return null;
+                })}
+                <div>
+                  <Forms.Button
+                    type="submit"
+                    variant={buttonVariant}
+                    onClick={this.onSubmit}
+                    style={{ marginRight: '16px' }}
```

Comment:
Instead of specifying margin you can wrap your buttons in `HorizontalGroup` layout like here https://github.com/grafana/grafana/pull/22395/files#diff-d2936ffa0c7be9e829013ec0d9f451a2R95